improver_tests/utilities/solar/test_DayNightMask.py

```diff
@@ -80,6 +82,8 @@ def test_basic_daynight_mask(self):
         self.assertEqual(result.units, unit.Unit("1"))
         self.assertEqual(result.data.min(), DayNightMask().night)
         self.assertEqual(result.data.max(), DayNightMask().night)
+        self.assertEqual(result.attributes["title"], "Day-Night mask")
+        self.assertEqual(result.attributes["institution"], "Met Office")
```

Comment:
It might be worth adding an assertion regarding the dtype of the resulting cube here.

Reply:
Done.